### PR TITLE
[core-aam] add new AX API mappings for colindextext/rowindextext

### PR DESCRIPTION
**Pull request by cookiecrook**
Closes https://github.com/w3c/core-aam/issues/53

# Test, Documentation and Implementation tracking
Once this PR has been reviewed and has consensus from the working group, tests should be written and issues should be opened on browsers. Add N/A and check when not applicable.

* [x] "author MUST" tests: n/a
* [x] "user agent MUST" tests: 
   * [x] not testable in WPT, so n/a
   * [ ] @spectranaut may wish to add platform mapping tests
* [ ] Browser implementations (link to issue or commit):
   * [ ] WebKit: [#257142](https://webkit.org/b/257142)
   * [ ] Gecko: [#1931415](https://bugzilla.mozilla.org/show_bug.cgi?id=1931415)
   * [x] Blink: [#378698863](https://issues.chromium.org/issues/378698863)
* Does this Mac-specific change need AT implementation? 
   * VoiceOver change is not prioritized b/c there is not a platform equivalent
* [x] Related APG Issue/PR: n/a
* [x] MDN Issue/PR: n/a

### CODE REVIEW

**Comment by cookiecrook** [comment]
@twilco @minorninth FYI

File: core-aam/index.html
Comment (cookiecrook) [inline core-aam/index.html:6375]: @twilco et al discussing whether we should double-down on having "ARIA" in the property name just because that was added to the related `AXARIAColumnIndex` property.... We may deprecate the ARIA-specific names in the future anyway.
```suggestion
                  <span class="property">Property: <code>AXColumnIndexText</code>: <code>&lt;value&gt;</code></span>
```

File: core-aam/index.html
Comment (cookiecrook) [inline core-aam/index.html:9212]: ditto.
```suggestion
                  <span class="property">Property: <code>AXRowIndexText</code>: <code>&lt;value&gt;</code></span>
```

**Comment by jnurthen** [comment]
@aleventhal @jcsteh @spectranaut We'd like to merge this before the CR snapshot - can we get this reviewed as soon as possible please. Thanks.

**Review by aleventhal — APPROVED** [review]

**Review by spectranaut — COMMENTED** [review]
@cookiecrook it seems like this new mapping is not implemented in any other the browsers, from a quick code search?

I added the implementation tracking, because it seems like we should be doing implementation tracking for this CL.  Before it lands we should have issues on the browsers to implement.

Because this still needs implementation, should it really block the CORE-AAM snapshot? Perhaps if you really want it in, we can at least make issues on the browsers before merging.

**Comment by aleventhal** [comment]
Funny you mention that, I'm working on a CL for it -- doesn't work yet:
https://chromium-review.googlesource.com/c/chromium/src/+/6012688

**Comment by cookiecrook** [comment]
Yes, it's new. So that ARIA WG doesn't have to publish a "TBD" in the spec.

**Comment by cookiecrook** [comment]
> should it really block the CORE-AAM snapshot? 

Otherwise I would expect it to be pulled or marked as At-Risk because this is a new feature without AT support on one platform, and without preceding vendor commitment. I outlined the concerns at:

- https://github.com/w3c/aria/issues/2361

**Comment by cookiecrook** [comment]
> Perhaps if you really want it in, we can at least make issues on the browsers before merging.

I will link the WebKit tracker. Sounds like Aaron has one for Blink already. I hesitate to file one for Gecko without acknowledgement from @jcsteh or @nmlapre that they want to do the mapping work before there is AT commitment.

**Comment by spectranaut** [comment]
> I have linked the WebKit tracker and the one Aaron filed for Blink. I hesitate to file one for Gecko without acknowledgement from @jcsteh or @nmlapre that they want to do the mapping work before there is AT commitment.

Sometimes I open issues on gecko and link back to the ARIA PR when I want to get review from firefox :) Personally I think it's fine to open an issue in this case, I don't think this change is controversial and besides it is coming from the API maintainers of AX API.

**Comment by spectranaut** [comment]
Discussed in today's ARIA working group meeting: https://www.w3.org/2024/11/14-aria-minutes.html#8070

We will add this to the AAM, and open issues on browsers, as eventually, probably, VoiceOver will do something with this value.

**Review by spectranaut — APPROVED** [review]
Per working group discussion, merging.

**Pull request merged**
